### PR TITLE
[Accessibility] Fixed the color contrast of the tutorial

### PR DESCRIPTION
**Pull request by veler**
Fixed the Orange color contrast of the tutorial by using the Microsoft color.

Related issue : [https://github.com/Microsoft/pxt/issues/2535](https://github.com/Microsoft/pxt/issues/2535)

untitled

### CODE REVIEW

**Comment by pelikhan** [comment]
It's very very red now. Isn't that too scary?

**Comment by abchatra** [comment]
Can we try a different color?

**Comment by microsoftsam** [comment]
This PR description doesn't make sense. You're updating the block color, yet you're showing the color change for the getting started button. The two should be independent.

**Comment by veler** [comment]
Hum, in fact, this change affect everything in orange : the block, the buttons (Getting started, Finish) and the orange top bar of the tutorial.
Do you want me to do the change only on the blocks? It would makes sense to me to change it on the buttons too as there is the same color contrast issue than on the block even if the accessibility team don't talk about the buttons explicitly.

**Comment by microsoftsam** [comment]
The issue in question here says the "Ok" and the text in orange in the getting started experience, ie the blocks are the issue here. They use an orange color that is non compliant given the text size and the orange color chosen.
You've already fixed the "Ok" button in a separate change, so I think this change should only fix the block text orange to use the MS color in all targets.
We are not able to change the micro:bit orange without Foundation consolation, let's discuss offline if that is necessary.

**Comment by veler** [comment]
I changed the color of the non-categorized inline block to use the MS orange color. I closed the PR related to micro:bit [https://github.com/Microsoft/pxt-microbit/pull/539](https://github.com/Microsoft/pxt-microbit/pull/539).
I updated the screenshot.

File: theme/themes/pxt/globals/site.variables [hunk 1]
```diff
@@ -18,6 +18,7 @@
 @grey:#505050;
 
 @greenTextColor: #107C10;
+@microsoftorange:#D83B01;
```
Comment (microsoftsam) [inline theme/themes/pxt/globals/site.variables]: No need to specify here. use the color directly. or rename to something more descriptive of it's use and place it at the top of tutorial.less

**Comment by veler** [comment]
TODO once merged : Do the same PR for Master, or cherry-pick to accessibility-master.

File: theme/tutorial.less [hunk 2]
```diff
@@ -5,6 +5,8 @@
 /* Reference import */
 @import (reference) "semantic.less";
 
+@inlineblockmicrosoftorange: #D83B01; // The official Microsoft orange color.
```
Comment (microsoftsam) [inline theme/tutorial.less]: change to inlineBlockColor..

Comment (microsoftsam) [reply theme/tutorial.less]: No need for comment.

**Pull request merged**